### PR TITLE
add callout to explain the difference

### PR DESCRIPTION
**Pull request by mirnawong1**
adding callout to blog post explaining the different btw dbt-cloud-cli and dbt cloud cli, powered by dbt cloud. refer to [slack thread](https://dbt-labs.slack.com/archives/C051TUB7S9W/p1698283391215239)

### CODE REVIEW

File: website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ date: 2022-05-03
 is_featured: true
 ---
 
+:::info Different from dbt Cloud CLI
+This is blog explains how to use the Python library, `dbt-cloud-cli` and create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a dbt Cloud-powered tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line.
```
Comment (matthewshaver) [inline website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md:16]: Minor grammatical fixes
```suggestion
This blog explains how to use the `dbt-cloud-cli` Python library to create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line.
```

Comment (matthewshaver) [inline website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md]: Minor grammatical fixes
```suggestion
This blog explains how to use the `dbt-cloud-cli` Python library to create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line.
```

Comment (mirnawong1) [inline website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md]:
```suggestion
This is blog explains how to use the Python library, `dbt-cloud-cli`, and create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a dbt Cloud-powered tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line.
```

**Review by matthewshaver — APPROVED** [review]

File: website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ date: 2022-05-03
 is_featured: true
 ---
 
+:::info Different from dbt Cloud CLI
+This is blog explains how to use the Python library, `dbt-cloud-cli`, and create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a dbt Cloud-powered tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line.
```
Comment (mirnawong1) [inline website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md]:
```suggestion
This blog explains how to use the `dbt-cloud-cli` Python library to create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line
```

File: website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ date: 2022-05-03
 is_featured: true
 ---
 
+:::info Different from dbt Cloud CLI
+This blog explains how to use the `dbt-cloud-cli` Python library to create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line
```
Comment (mirnawong1) [inline website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md]:
```suggestion
This blog explains how to use the `dbt-cloud-cli` Python library to create a data catalog app with dbt Cloud artifacts. This is different from the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), a tool that allows you to run dbt commands against your dbt Cloud development environment from your local command line.
```

**Pull request merged**